Scripts/DBotBuildPhishingClassifier/DBotBuildPhishingClassifier.py

```diff
@@ -0,0 +1,33 @@
+from CommonServerPython import *
```

Comment:
need to import here base64 in order to execute the script without commonserver, as done in the feature